components/table/Sidebar.jsx

```diff
@@ -0,0 +1,48 @@
+import React, { useRef } from 'react'
+
+import tableClassNames from './index.css'
+
+import { CloseIcon } from 'design'
+
+class Sidebar extends React.Component {
+  constructor(props) {
+    super(props)
+    const { children } = props
+    if (children) React.Children.only(children)
+  }
+
+  render() {
+    const { isOpen, children, ...passProps } = this.props
+    return (
+      <div className={tableClassNames.sidebar}>
+        {children && React.cloneElement(children, passProps)}
+      </div>
+    )
+  }
+}
+
+Sidebar.displayName = 'TableSidebar'
```

Comment:
It's not TableSidebar at all. It's a separate component not related to table. Maybe something like _DetailsPane_

Reply:
This line is just relic cod and it's gonna be removed in https://github.com/oacore/dashboard/pull/196. 

>  It's a separate component not related to the table. 

That's questionable. styling is tight up with a table component.

Reply:
Yes, you may consider it questionable. This is the only place you see the component and it's tightly coupled to the Table. However, as a software engineer you are the designer of the application too. To my mind, if there is no reason to tie 2 components, why do it just because it looks so. Software design should not reflect UI design 1-by-1. UI design may change often but software should not change as often.